presto-docs/src/main/sphinx/connector/phoenix.rst

```diff
@@ -0,0 +1,135 @@
+===================
+Phoenix Connector
+===================
+
+The Phoenix connector allows querying data stored in Phoenix.
```

Comment:
can you please add a link to it?

Reply:
I do not understand your question. Do you mean I need to be added to the `` connector.rst`` file like ``connector/phoenix``? Then I did it.

Reply:
I wanted to add a link to https://phoenix.apache.org/. Please nevermind this comment, I noticed that none of other connector does that.